### ISSUE TITLE
Social button keyboard focus

### ISSUE THREAD

**Issue by khang-nd** [ntnyq/vuepress-plugin-social-share#56]
Hi mate, thanks for making this awesome plugin, extremely useful and a time saver.

I would like to point out a minor issue with the social buttons, there is no style when they are focused using the `Tab` key. Apparently it is due to this css line: https://github.com/ntnyq/vuepress-plugin-social-share/blob/53cbed61ceb3a5478e9c1cb1c4d39b7002c9a1cb/src/client/style.css#L41
Not sure what the intention was when you removed the `outline` but I believe it's not a good a11y practice to have no focusing style.
